server/api.js

```diff
@@ -35,34 +35,35 @@ const api = module.exports = express.Router({
 
 api.get('/', (req, res) => res.redirect('/docs/api'));
 
-api.post('/register', GUARD, REGISTER_USER, (req, res) => res.sendStatus(200));
+api.post('/known', GUARD, KNOWN_USER, (req, res) => res.sendStatus(200));
```

Comment:
rename GUARD to "google_gives_us_a_user" - to make it clear that this isn't "logged into LIMA" just "ready to give lima login details"